alpha-blog/app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,51 @@
+@import 'bootstrap/dist/css/bootstrap';
```

Comment:
Can you try `@use` ?
if it works, change it to that. To understand why, google pls